docs/Change_Log.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+#### April 3, 2023
+
+## Interaction Channel Data
+
+Interactions now contain a `channel` field which is a partial channel object and guaranteed to contain `id` and `type`. We recommend that you begin using this channel field to identify the source channel of the interaction, and may deprecate the existing `channel_id` field in the future. See the [interaction documentation](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object) for more details.
```

Comment:
It would be convenient to also have `parent_id` present, so you can determine your permissions without needing to make a rest request in case you don't cache threads. Also, if you're missing permission to read a thread, you can't make rest requests to get more info anyway so the detail given here would be very helpful

Reply:
They should have `parent_id`, along with a number of other fields. Better documentation of partial objects is definitely something we want to improve in the future.

Reply:
currently partial objects in interactions list all fields included, can that be done here too?

![image](https://user-images.githubusercontent.com/11778454/229631398-1c7cadac-df7d-4964-9a1c-ae39e4394a1d.png)